src/core/utils/color/ColorValue.ts

```diff
@@ -147,36 +232,114 @@ export class ColorValue {
     return scratchUInt32[0];
   }
 
-  /** Create a ColorValue from its 0xTTBBGGRR representation. */
-  public static fromTbgr(tbgr: number): ColorValue {
-    return new ColorValue(tbgr);
-  }
-
   /** Create a ColorValue from a string representation. The following representations are supported:
    * *"rgb(255,0,0)"*
    * *"rgba(255,0,0,.2)"*
    * *"rgb(100%,0%,0%)"*
    * *"hsl(120,50%,50%)"*
-   * *"#rrbbgg"*
-   * *"blanchedAlmond"* (see possible values from [[ColorByName]]). Case-insensitive.
+   * *"#rrggbb"*
    */
   public static fromString(val: string): ColorValue {
     return this.fromTbgr(this.computeTbgrFromString(val));
   }
 
+  /** Create a ColorValue from hue, saturation, lightness values.  */
+  public static fromHSL(hsl: HslColor): ColorValue {
```

Comment:
Now thinking if we really need all these `from...`. You can just use `ColorValue.create`

Reply:
What if I just make them private to keep the create method from getting too big?